src/lightmap.cpp

```diff
@@ -331,6 +331,15 @@ void map::generate_lightmap()
                 }
             }
         }
+        // atomic light is always on
+        std::vector<int> light_indices = v->all_parts_with_feature(VPFLAG_ATOMIC_LIGHT);
```

Comment:
Ahh, and the other _on flags are implicitly _exists flags... You're right, I do need that here. But I'm not sure the right place to cache that flag.